web/src/main/webapp/resources/js/atsy-statehistory.js

```diff
@@ -10,6 +10,18 @@ function isNewState() {
   return queryStr.match(/state\=/);
 }
 
+function escapeXml(unsafe) {
```

Comment:
A method for escaping already exists in atsy.js called escapeHTMLFormatter. Please use the existing escapeHTMLFormatter method instead of duplicate solution for this escaping issue.